ui/StatusQ/src/wallet/managetokensmodel.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <QAbstractListModel>
+#include <QLoggingCategory>
+
+Q_DECLARE_LOGGING_CATEGORY(manageTokens)
+
+namespace
+{
+const auto kSymbolRoleName = QByteArrayLiteral("symbol");
+const auto kNameRoleName = QByteArrayLiteral("name");
+const auto kCommunityIdRoleName = QByteArrayLiteral("communityId");
+const auto kCommunityNameRoleName = QByteArrayLiteral("communityName");
+const auto kCommunityImageRoleName = QByteArrayLiteral("communityImage");
+const auto kCollectionUidRoleName = QByteArrayLiteral("collectionUid");
+const auto kCollectionNameRoleName = QByteArrayLiteral("collectionName");
+const auto kEnabledNetworkBalanceRoleName = QByteArrayLiteral("enabledNetworkBalance");
+const auto kEnabledNetworkCurrencyBalanceRoleName = QByteArrayLiteral("enabledNetworkCurrencyBalance");
+const auto kCustomSortOrderNoRoleName = QByteArrayLiteral("customSortOrderNo");
+const auto kTokenImageRoleName = QByteArrayLiteral("imageUrl");
+} // namespace
+
+struct TokenData {
```

Comment:
That's the `symbol` for now (explained above)

Reply:
Ok, we can keep the `symbol` on this stage but it's a pending task to update it to use the `key`, ok?? Otherwise we can be confused again (I understand the confusion in here, since before we were using the _symbol_ in some cases) and we are trying to unify things!

Reply:
Yeah, it's definitely a good thing to have, a unique id. Pls file a separate issue assigned to me